gravitee-apim-definition/gravitee-apim-definition-model/src/main/java/io/gravitee/definition/model/v4/listener/http/HttpListener.java

```diff
@@ -63,7 +63,13 @@ public HttpListener() {
     public void setPathMappings(final Set<String> pathMappings) {
         this.pathMappings = pathMappings;
         if (pathMappings != null) {
-            setPathMappingsPattern(pathMappings.stream().collect(toMap(pathMapping -> pathMapping, Pattern::compile)));
+            setPathMappingsPattern(
+                pathMappings
+                    .stream()
+                    .collect(
+                        toMap(pathMapping -> pathMapping, pathMapping -> Pattern.compile(pathMapping.replaceAll(":\\w*", "[^\\/]*") + "/*"))
```

Comment:
I'm not sure to understand the `pathMapping.replaceAll(":\\w*", "[^\\/]*") + "/*")`

Is it made to replace such path `/:id/example/:foo/:bar/something` with a corresponding pattern that matches it? 
If so, I don't understand well the last part with the ` + "/*"` which expects 0 or n '/' ? 🤔

Reply:
It's not directly related to the issue but it was to align with what was done in v2 definition with ApiDeserializer https://github.com/gravitee-io/gravitee-api-management/blob/b08ca348b3a10cd518e3238b8356d9769eb6cd8d/gravitee-apim-definition/gravitee-apim-definition-jackson/src/main/java/io/gravitee/definition/jackson/datatype/api/deser/ApiDeserializer.java#L215